ion-c-sys/src/reader.rs

```diff
@@ -122,13 +80,7 @@ impl<'a> IonCReaderHandle<'a> {
     /// # Ok(())
     /// # }
     /// ```
-    #[inline]
-    pub fn depth(&self) -> IonCResult<i32> {
-        let mut depth = 0;
-        ionc!(ion_reader_get_depth(self.reader, &mut depth))?;
-
-        Ok(depth)
-    }
+    fn depth(&self) -> IonCResult<i32>;
```

Comment:
They do not--Ion C has a philosophical inclination to rarely use unsigned integers to avoid some of the various arithmetic problems that come from mixing signed and unsigned arithmetic.

Reply:
Another place this Ion C idiom pops up in is the various things that return or take `SIZE` which is `i32`, we have a bunch of code that just does `try_into()?` to do those conversions and it is annoying.